nix-tools-release.nix

```diff
@@ -0,0 +1,69 @@
+commonLib: # the iohk-nix commonLib
+{ packages ? []
+, required-name ? "required"
+, required-targets ? (jobsets: [])
+, config ? {}
+, rev ? "abcdef"
+, package-set-path # usually import ./. {}
```

Comment:
is it a path or the imported thing?

Reply:
It's actually the path, and the comment is wrong. Good catch!